JumpscaleLibs/sal/reservation_chatflow/reservation_chatflow.py

```diff
@@ -586,14 +586,25 @@ def payments_show(self, bot, reservation_create_resp, currency):
             wallet_names.append(w)
         wallet_names.append("3bot app")
 
+        explorer = j.clients.explorer.default
+        payment_details = ""
+        if currency == 'FreeTFT':
+            payment_details += f"Transaction Fees --------- {0.1} {currency}<br>"
```

Comment:
is it possible to align the the start of each price to be under each other like this : 
Farmer a -------- X.XX
Farmerbbbb ---- X.XX
Farmer cccc----- X.XX